client/src/Features/AssignmentLinks/AssignmentLinkEditor.tsx

```diff
@@ -18,6 +18,22 @@ interface AssignmentLinkEditorProps {
 }
 export type AssignmentLinkEditorStyles = SimpleComponentStyles<'root'>;
 
+const ValidateTextLength = (limit: number) => (text: string | undefined) => (text && text?.length < limit) ? "The entered text is too long." : "";
+export const ValidateTitleLength = ValidateTextLength(500);
+export const ValidateDescLength = ValidateTextLength(1000);
+
+export const isValidURL = (value: string | undefined) => {

```

Comment:
Should we also check for cases that the url is valid **HTTPUrl** 🤔 ?

Reply:
Added a check for the `url.protocol` to the client, and the backend.